server/routerlicious/packages/services-telemetry/src/lumberEventNames.ts

```diff
@@ -50,4 +50,6 @@ export enum LumberEventName {
 	ConnectionCountPerNode = "ConnectionCountPerNode",
 	RestoreFromCheckpoint = "RestoreFromCheckpoint",
 	ReprocessOps = "ReprocessOps",
+	GetDeltasFromDb = "GetDeltasFromDb",
+	GetDeltasFromStorage = "GetDeltasFromStorage",
```

Comment:
The changes make sense for OSS, however, FRS is not using this piece of logic. Don't see the benefits the metrics you added here.

FRS has a file called fallbackDeltaService.ts, which has almost all the metric/status you need.

Reply:
I think adding this metric here would help us in analysing the time taken for the db call during the whole get deltas API duration. We can still do that outside in the fallbackDeltaService, but I thought this would be cleaner since we also have a similar metric for GetDeltasFromFallback to capture that duration.

If amount of logs is a concern, I can alternately add the duration for both GetDeltasFromDb as well as GetDeltasFromFallback to the overall GetDeltaObjectStoreFallback metric and remove this metric from OSS then - what do you think?

I will remove the GetDeltasFromStorage metric since that is obsolete and not needed.

Reply:
It is up to you, I prefer to use single metrics instead of double the metrics due to
1. The amount of logs, we currently already have too many logs crashing geneva buffer, which lead to lost logs. Also it leads to kusto ingestion delay. @hedasilv do you have any concerns?
2. it is difficult to correlate the two metric from the same call. But if you can figure out a better way to correlate, I think it is fine.

Reply:
1. I see the benefit of adding a new metric here to analyse the GetDeltasFromDb trend, its faster to check and plot the direct duration of a metric instead of extracting that from the properties of another metric. 
2. However I also see the benefit of adding this latency to the overall GetDeltaObjectStoreFallback's metric properties since its easier to check/correlate for a single call.

I am going to do #2 anyways since we already have that metric. Since number of logs is a concern here, lets not do #1 for now and will reconsider if needed after adding #2.